src/app/components/login/login.component.ts

```diff
@@ -0,0 +1,72 @@
+import { Component, inject } from '@angular/core';
+import {
+  FormControl,
+  FormGroup,
+  ReactiveFormsModule,
+  Validators,
+} from '@angular/forms';
+import { MatFormFieldModule } from '@angular/material/form-field';
+import { MatInputModule } from '@angular/material/input';
+import { MatIconModule } from '@angular/material/icon';
+import { MatButtonModule } from '@angular/material/button';
+import { AuthService } from '../../services/auth.service';
+import { MatDividerModule } from '@angular/material/divider';
+import { Router } from '@angular/router';
+import { MatCardContent, MatCardModule } from '@angular/material/card';
+
+@Component({
+  selector: 'app-login',
+  standalone: true,
+  imports: [
+    ReactiveFormsModule,
+    MatFormFieldModule,
+    MatInputModule,
+    MatIconModule,
+    MatButtonModule,
+    MatDividerModule,
+    MatCardModule,
+    MatCardContent,
+  ],
+  templateUrl: './login.component.html',
+  styleUrl: './login.component.scss',
+})
+export class LoginComponent {
+  authService = inject(AuthService);
+  router = inject(Router);
+
+  loginForm = new FormGroup({
+    email: new FormControl('', [Validators.required, Validators.email]),
+    password: new FormControl('', [
+      Validators.required,
+      Validators.minLength(8),
+    ]),
+  });
+
+  hide = true;
+
+  onSubmit() {
+    if (this.loginForm.value.email && this.loginForm.value.password)
+      this.authService
+        .login(this.loginForm.value.email, this.loginForm.value.password)
+        .then(() => this.router.navigate(['']))
+        .catch(() =>
```

Comment:
Quick suggestion, you could use a notification to alert users of the error as it is important for them to know what went wrong so they could rectify the mistake if it was from their side. Same goes for the other auth functions.